Sources/_MatchingEngine/Regex/AST/AST.swift

```diff
@@ -153,7 +153,7 @@ extension AST {
     public init(_ mems: [AST.Node], pipes: [SourceLocation]) {
       // An alternation must have at least two branches (though the branches
       // may be empty AST nodes), and n - 1 pipes.
-      precondition(mems.count >= 2)
+      // precondition(mems.count >= 2)
```

Comment:
What's the story here?

Reply:
This had to be uncommented before we had `DSLTree`, but now I can revert this part and make this be DSL-specific.

Reply:
Done.